docs/how_to_guides.md

```diff
@@ -1,6 +1,24 @@
 ---
 id: how_to_guides
-title: How to guides
+title: How to consume HackIT's MicroFrontend
 ---
+## Introduction
+
+We have created a video that gives developers a good understanding of how buld microfronted using HackIT's tools.
+
+The video below will go through the following sections:
+- Mtfh-cli setup
+- Capability of cli
+- Running mtfh-cli on localhost
+- Create new Microfrontend
+- Integrate new microfrontend with root config
+- Common microfrontend
+- Consume api in microfrontend 
+
+
+##  Video Tutorial
+Note: To go back a forth between the videos please use the playlist button available
+<figure class="video-container">
+<iframe width="100%" src="https://www.youtube.com/embed/videoseries?list=PL1mVZlA7eC8Qd99uNCXsd7NseKrp1fHA1" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Not sure, but this might not work exactly like it does on the playbook because the CSS hasn't been set up the same way. The 100% width might do something weird.

See [this PR](https://github.com/LBHackney-IT/API-Playbook/pull/31) on the API Playbook for the CSS changes, you could implement them here or just leave the hardcoded width/heights

Reply:
Never mind, just saw that was moved over from the API Playbook, all good :)